src/pages/User/components/PullRequests/PullRequest/PullRequestInfo.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { formatDate } from '../../../../../utils/utils';
+import { formatDate } from '../../../../../utils/date';
```

Comment:
you can try absolute import to remove the chain of `../`

Reply:
https://medium.com/hackernoon/absolute-imports-with-create-react-app-4c6cfb66c35d you can follow this article